graphql2/graphqlapp/alertlog.go

```diff
@@ -0,0 +1,129 @@
+package graphqlapp
+
+import (
+	"context"
+	"github.com/pkg/errors"
+	"github.com/target/goalert/alert"
+	"github.com/target/goalert/alert/log"
+	"github.com/target/goalert/graphql2"
+	"github.com/target/goalert/search"
+	"github.com/target/goalert/validation/validate"
+)
+
+type AlertLog App
+
+func (a *App) AlertLog() graphql2.AlertLogResolver { return (*AlertLog)(a) }
+
+func (l *AlertLog) Event(ctx context.Context, log *alert.Log) (graphql2.Event, error) {
+	return graphql2.Event(log.Event), nil
+}
+
+func (l *AlertLog) AlertID(ctx context.Context, log *alert.Log) (int, error) {
+	var s alertlog.SearchOptions
+	s.Start = log.Timestamp
+	s.Event = alertlog.Type(log.Event)
+	s.Limit = 1
+
+	entries, _, err := l.LogStore.Search(ctx, &s)
```

Comment:
Is there a way we can get this (e.g. from the `alert.Log`) without doing a query?